release-tools/src/main/java/org/springframework/data/release/jira/IssueTrackerConfiguration.java

```diff
@@ -77,12 +79,13 @@ public RestTemplate restTemplate() {
 	}
 
 	@Bean
-	public Jira jira(Logger logger) {
-		return new Jira(restTemplate(), logger);
+	public Jira jira(Logger logger, @Value("${jira.url}") String jiraUrl) {
```

Comment:
Should we rather use Boot's configuration properties support? The codebase uses that in every other place already.